app/MorningProgramScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { Dimensions, ScrollView, StyleSheet, Text, View } from 'react-native';
+import Carousel from 'react-native-reanimated-carousel';
+
+//importing morning program data
+import { MORNING_PROGRAM_INFO, MorningProgram } from './data/MorningProgramInfo';
+import QuestionIcon from './icons/QuestionIcon';
+
+const { width: screenWidth } = Dimensions.get('window');
+
+const MorningProgramScreen: React.FC = () => {
+  const [activeSlide, setActiveSlide] = useState(0);
+  const morningProgramArray = Object.values(MORNING_PROGRAM_INFO);
+
+  const renderItem = ({ item }: { item: MorningProgram }) => (
+    <ScrollView style={styles.item}>
+      <Text style={styles.title}>{item.title}</Text>
+      <Text style={styles.subtitle}> What's Happening: </Text>
+      <Text style={styles.body}> {item.description}</Text>
+      {item.helpInfo && (
+        <View style={styles.standoutText}>
+          <View style={styles.row}>
+            <QuestionIcon />
+            <Text style={styles.helpSubtitle}>How you can help! </Text>
+          </View>
+          <Text style={styles.helpBody}>{item.helpInfo}</Text>
+        </View>
+      )}
+    </ScrollView>
+  );
+
+  return (
+    <View style={styles.container}>
+      <Carousel
+        data={morningProgramArray}
+        renderItem={renderItem}
+        width={screenWidth}
+        height={670}
```

Comment:
Can you decrease the height of the progress doc padding? Right now it's blocking the bottom of the screen for some screens that require scrolling

Reply:
![Uploading image.jpeg…]()